packages/plans-grid-next/src/components/features-grid/client-logo-list/style.scss

```diff
@@ -2,7 +2,7 @@
  * Logo list styles
  */
 
-.client-logo-list {
+.plans-grid-next-features-grid-client-logo-list {
```

Comment:
Again with regards to long classnames: does it make sense to have a parent class, and then children with shorter classnames? This could also be done as a follow-up.

Reply:
I fixed the modifiers to Calypso-style since it was using traditional BEM style: b45809b7bdad0af2496a192facb4fe83728a6adf